v2/pkg/genruntime/admissions.go

```diff
@@ -30,6 +30,7 @@ type Defaulter interface {
 	CustomDefault()
 }
 
+// TODO: Fix here
```

Comment:
Either address this TODO and remove it, or expand it so someone else will know what needs fixing.

Reply:
Doh, didn't intend to leave this.

Reply:
This was just a note to myself that this was where I needed to actually make the fix (when I was investigating how to fix it).

Removed.